			   READLINE PATCH REPORT
			   =====================

Readline-Release: 6.2
Patch-ID: readline62-002

Bug-Reported-by:	Vincent Sheffer <vince.sheffer@apisphere.com>
Bug-Reference-ID:	<F13C1C4F-C44C-4071-BFED-4BB6D13CF92F@apisphere.com>
Bug-Reference-URL:	https://lists.gnu.org/archive/html/bug-readline/2011-08/msg00000.html

Bug-Description:

The readline shared library helper script needs to be updated for Mac OS X
10.7 (Lion, darwin11).

Index: readline6-6.2/patchlevel
===================================================================
--- readline6-6.2.orig/patchlevel	2010-01-14 15:15:52.000000000 +0000
+++ readline6-6.2/patchlevel	2012-09-17 12:14:26.063451590 +0000
@@ -1,3 +1,3 @@
 # Do not edit -- exists only for use by patch
 
-1
+2
Index: readline6-6.2/support/shobj-conf
===================================================================
--- readline6-6.2.orig/support/shobj-conf	2012-09-17 12:13:44.203470014 +0000
+++ readline6-6.2/support/shobj-conf	2012-09-17 12:14:26.063451590 +0000
@@ -157,7 +157,7 @@
 	;;
 
 # Darwin/MacOS X
-darwin[89]*|darwin10*)
+darwin[89]*|darwin1[012]*)
 	SHOBJ_STATUS=supported
 	SHLIB_STATUS=supported
 	
@@ -186,7 +186,7 @@
 	SHLIB_LIBSUFF='dylib'
 
 	case "${host_os}" in
-	darwin[789]*|darwin10*)	SHOBJ_LDFLAGS=''
+	darwin[789]*|darwin1[012]*)	SHOBJ_LDFLAGS=''
 			SHLIB_XLDFLAGS='-dynamiclib -arch_only `/usr/bin/arch` -install_name $(libdir)/$@ -current_version $(SHLIB_MAJOR)$(SHLIB_MINOR) -compatibility_version $(SHLIB_MAJOR) -v'
 			;;
 	*)		SHOBJ_LDFLAGS='-dynamic'
